Doc/library/crypt.rst

```diff
@@ -111,7 +111,7 @@ The :mod:`crypt` module defines the following functions:
    .. index:: single: crypt(3)
 
    Since a few :manpage:`crypt(3)` extensions allow different values, with
-   different sizes in the *salt*, it is recommended to use  the full crypted
+   different sizes in the *salt*, it is recommended to use the full crypted
```

Comment:
Sorry, it's not worth touching the line to remove one extra space. Every change makes looking through git history a bit more time consuming so trivial changes that don't impact the reader should be avoided.

```suggestion
   different sizes in the *salt*, it is recommended to use  the full crypted
```